docs/pages/grid.md

```diff
@@ -10,7 +10,7 @@ tags:
 
 ## Basics
 
-Start by adding an element with a class of `.row`. This will create a horizontal block to contain vertical columns. Then add elements with a `.column` class within that row. You can use `.column` or `.columns`&mdash;the only difference is grammar. Specify the widths of each column with the `.small-#`, `.medium-#`, and `.large-#` classes.
+Start by adding an element with a class of `.row`. This will create a horizontal block to contain vertical columns. Then add elements with a `.column` class within that row. You can use `.column` or its <a href="http://localhost:3000/grid.html#sass-reference">alias</a> `.columns`&mdash;the only difference is grammar. Specify the widths of each column with the `.small-#`, `.medium-#`, and `.large-#` classes.
```

Comment:
So from 3,8% and 5,2%. It is not negligible.

You can add: `Disabling the alias can reduce the Foundation CSS file size from 3 to 5%. It is recommended if the alias is not used.`